test/execute/README.md

```diff
@@ -0,0 +1,20 @@
+Dredd *execute* tests
+=====================
+
+There is one subdirectory per test.
+
+Each subdirectory must contain:
+
+- `harness.c(c)` - a C or C++ file with a `main`, that should print a single line of output.
+
+- `tomutate.c(c)` - a C or C++ file that will be mutated. The harness should call functions of this file.
+
+- `original.txt` - A file containing the single line of output that should be printed by the harness when no mutants are enabled.
```

Comment:
Is "single line"  required?

I noted that the the check is performed by extracting output
```
original_result.stdout.decode('utf-8')
```
in `check_one_execute_test.py`. which should works for multiple line?

Reply:
Moreover, the expected output is extracted by
```
open(test_directory / 'original.txt', 'r').read()
```
which should read more than a line.